cli/commands/login.go

```diff
@@ -54,7 +54,11 @@ func (cmd *Login) Metadata() command_metadata.CommandMetadata {
 		Name:        "login",
 		Description: "Login to brooklyn",
 		Usage:       "BROOKLYN_NAME login URL [USER [PASSWORD]]",
-		Flags:       []cli.Flag{cli.BoolFlag{Name: "skipSslChecks", Usage: "Skip SSL Checks"}},
+		Flags:       []cli.Flag{
+			cli.BoolFlag{Name: "skipSslChecks", Usage: "Skip SSL Checks"},
+			cli.BoolFlag{Name: "noCredentials", Usage: "No user/password needed"},
```

Comment:
`noCredentials` is a bad name, you are still supplying credentials, just not basic auth credentials. See my general remarks for a suggested alternative.